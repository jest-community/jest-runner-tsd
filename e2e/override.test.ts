import runJest from './runJest';

test('overrides "strict": true in tsconfig', async () => {
  expect(await runJest('override-tsconfig')).toMatchSnapshot();
});
