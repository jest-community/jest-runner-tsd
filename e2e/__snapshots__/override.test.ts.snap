// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`overrides "strict": true in tsconfig 1`] = `
"PASS e2e/__fixtures__/override-tsconfig/index.test.ts
  âœ“ tsd typecheck
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
